src/referral_maintenance.rs

```diff
@@ -56,36 +60,41 @@ pub async fn maintenance_tasks(
         }
     };
     // 1.2: Load app_data from solvable_orders api
+    tracing::info!("Loading app data from solvable_orders api");
     match load_current_app_data_of_solvable_orders().await {
         Ok(vec_with_app_data) => vec_with_all_app_data.extend(vec_with_app_data),
         Err(err) => tracing::warn!("error while downloading the solvable orders: {:}", err),
     };
     // 2nd step: Store app_data in ReferralStore, if not yet existing
+    tracing::info!("Store app_data in ReferralStore, if not yet existing");
     for app_data in vec_with_all_app_data {
         {
             let mut guard = db.0.lock().map_err(|_| anyhow!("Mutex poisoned"))?;
             guard
                 .app_data
                 .entry(app_data)
-                .or_insert(Referral::TryToFetchXTimes(retrys_for_ipfs_file_fetching));
+                .or_insert(AppDataJson::TryToFetchXTimes(retrys_for_ipfs_file_fetching));
         }
     }
-    // 3st step: get all uninitialized referrals
+    // 3rd step: get all uninitialized referrals
+    tracing::info!("get all uninitialized referrals");
```

Comment:
Since the previous messages started with upper letter, maybe we can do it here as well?